osu.Game/OsuGameBase.cs

```diff
@@ -419,7 +419,7 @@ public override void SetHost(GameHost host)
         /// Use to programatically exit the game as if the user was triggering via alt-f4.
         /// Will keep persisting until an exit occurs (exit may be blocked multiple times).
```

Comment:
Would reword to give the idea that `GracefullyExit` could be overriden, or maybe remove this line entirely.
```suggestion
        /// By default, this will keep persisting until an exit occurs (exit may be blocked multiple times).
```